pilot/cmd/pilot-agent/metrics/metrics.go

```diff
@@ -12,25 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package status
+package metrics
 
-import "istio.io/pkg/monitoring"
+import (
+	"time"
+
+	"istio.io/pkg/monitoring"
+)
 
 var (
 	typeTag = monitoring.MustCreateLabel("type")
 
+	// StartupTime measures the time it takes for the agent to get ready Note: This
+	// is dependant on readiness probes. This means our granularity is correlated to
+	// the probing interval.
+	startupTime = monitoring.NewGauge(
+		"startup_time",
```

Comment:
instead of just `_time`, can you make this `startup_duration_seconds` or whatever the appropriate unit is.

Reply:
yep